engine/src/main/java/org/hibernate/validator/internal/metadata/core/MetaConstraint.java

```diff
@@ -46,22 +47,22 @@
 	private final ConstraintLocation location;
 
 	/**
-	 * The {@link ValueExtractor} used to extract the value for validation.
+	 * The {@link ValueExtractor}s used to extract the value for validation.
 	 */
-	private final ValueExtractorDescriptor valueExtractorDescriptor;
+	private final List<ValueExtractorDescriptor> valueExtractorDescriptors;
```

Comment:
Can you add an explantion why this is a list now? I reckon it's to navigate from the outermost container to the innermost?